library/OrderCloud.Catalyst/Integrations/Implementations/Vertex/Mappers/VertexRequestMapper.cs

```diff
@@ -0,0 +1,104 @@
+﻿using OrderCloud.SDK;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+
+namespace OrderCloud.Catalyst
+{
+	public static class VertexRequestMapper
+	{
+		public static VertexCalculateTaxRequest ToVertexCalculateTaxRequest(this OrderWorksheet order, List<OrderPromotion> promosOnOrder, string companyCode, VertexSaleMessageType type)
+		{
+			var itemLines = order.LineItems.Select(li => ToVertexLineItem(li));
+			var shippingLines = order.ShipEstimateResponse.ShipEstimates.Select(se =>
```

Comment:
Using `var` is a pattern that's pretty engrained in me from all the C# projects at four51. I do see how it makes reviewing code hard if you're not viewing in the VS edit (where hovering shows you the type). 

I'm willing to consider this. It would be a rather large code change throughout the project. Do you know what the OC platform code does? That's often what we model these other tools on.

Reply:
My personal opinion is to use it only if the type isn't immediately clear. Adds a lot of clutter and is unnecessarily verbose in many situations

ie:
```
Car car = new Car();
```